test-skripsi/test-1.py

```diff
@@ -0,0 +1,41 @@
+
+def calculate_discount(price, discount):
+    discounted_price = price * discount
+    return discounted_price
+
+def factorial(n):
+    if n == 1:
+        return 1
+    else:
+        return n * factorial(n)  
```

Comment:
The `factorial` function will result in a `RecursionError` due to infinite recursion. The recursive call should be `factorial(n-1)`, not `factorial(n)`. This decrement will ensure the recursion base case is eventually reached.